### PR TITLE
initial commit for vm import docs

### PR DESCRIPTION
**Pull request by ibrokethecloud**
PR introduces documentation updates for the vm-import-controller addon.

related to: https://github.com/harvester/harvester/issues/2274

### CODE REVIEW

File: docs/advanced/vmimport.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+By default the vm-import-controller leverages ephemeral storage, which is mounted from /var/lib/kubelet, however for large VM imports users should configure the addon with a PVC backed storage provided by longhorn.
+
+![](/img/v1.1/vm-import-controller/ConfigureAddon.png)
+
+## vm-import-controller
+
+Currently the following source providers will be supported:
+* vmware
+* openstack
```
Comment (bk201) [inline docs/advanced/vmimport.md:24]: Could we also add which VMWare and OpenStack we have already verified? Thanks!

File: docs/advanced/vmimport.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  namespace: default
+spec:
+  endpoint: "https://devstack/identity"
+  region: "RegionOne"
+  credentials:
+    name: devstack-credentials
+    namespace: default
+```
+
+The secret contains the credentials for the vcenter endpoint:
```
Comment (bk201) [inline docs/advanced/vmimport.md:87]:
```suggestion
The secret contains the credentials for the OpenStack endpoint:
```

File: docs/advanced/vmimport.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+  - sourceNetwork: "public"
+    destinationNetwork: "default/vlan2"
+  sourceCluster: 
+    name: devstack
+    namespace: default
+    kind: OpenstackSource
+    apiVersion: migration.harvesterhci.io/v1beta1
+```
+
+*NOTE:* Openstack allows users to have multiple instances with the same name. In such a scenario the users are advised to use the Instance ID. The reconcile logic tries to perform a lookup from name to ID when a name is used.
```
Comment (bk201) [inline docs/advanced/vmimport.md:176]: We can also use the `:::note` admonition.

File: docs/advanced/vmimport.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+```shell
+$ kubectl get virtualmachineimport.migration
+NAME                    STATUS
+alpine-export-test      virtualMachineRunning
+openstack-cirros-test   virtualMachineRunning
+
+```
+
+Similarly, users can define a VirtualMachineImport for OpenStack source as well:
```
Comment (vickyhella) [inline docs/advanced/vmimport.md:160]:
```suggestion
Similarly, users can define a VirtualMachineImport for the OpenStack source as well:
```

File: docs/advanced/vmimport.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+    destinationNetwork: "default/vlan2"
+  sourceCluster: 
+    name: devstack
+    namespace: default
+    kind: OpenstackSource
+    apiVersion: migration.harvesterhci.io/v1beta1
+```
+
+:::note 
+OpenStack allows users to have multiple instances with the same name. In such a scenario the users are advised to use the Instance ID. The reconcile logic tries to perform a lookup from name to ID when a name is used.
```
Comment (vickyhella) [inline docs/advanced/vmimport.md:183]:
```suggestion
OpenStack allows users to have multiple instances with the same name. In such a scenario, users are advised to use the Instance ID. The reconciliation logic tries to perform a name-to-ID lookup when a name is used.
```

**Review by vickyhella — COMMENTED** [review]
Just a few suggestions, the rest LGTM

**Review by bk201 — APPROVED** [review]
LGTM, thanks for addressing some requests.

**Review by guangbochen — APPROVED** [review]

**Pull request merged**